flink-connector-oceanbase/src/main/java/com/oceanbase/connector/flink/connection/OceanBaseConnectionProvider.java

```diff
@@ -95,6 +97,8 @@ protected void init() {
                     druidDataSource.setUsername(options.getUsername());
                     druidDataSource.setPassword(options.getPassword());
                     druidDataSource.setDriverClassName(options.getDriverClassName());
+                    druidDataSource.setConnectProperties(
```

Comment:
I'm not sure how `setConnectProperties` works, it's better to set these properties directly in the jdbcUrl, and print it in log after the modification.

Reply:
Druid will use the Properties set by the `setConnectProperties` method to create a Connection. see `com.alibaba.druid.pool.DruidAbstractDataSource#createPhysicalConnection()`.

This can avoid dealing with complex string concatenation logic and increase the robustness of the program.